assets/queries/terraform/azure/network_interfaces_dont_use_public_IPs/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "c1573577-e494-4417-8854-7e119368dc8b",
+  "queryName": "Network Interfaces Don't Use Public IPs",
```

change directory name to `network_interface_with_public_ip`

```suggestion
  "queryName": "Network Interface With Public IP",
```